### PR TITLE
Add the schedule

### PR DESCRIPTION
**Pull request by anah**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,6 +81,167 @@ <h5 class="sponsor-heading">sponsored by</h5>
...
+                  <th class="room"><h3>Workshop Room</h3></th>
+                </tr>
+              </thead>
+              <tbody>
+              <tr>
+                <th scope="row">8:00</th>
+                <td colspan="2" class="red">Registration</td>
+              </tr>
+              <tr>
+                <th scope="row">9:00</th>
```
Comment (nailor) [inline index.html]: I think @haard was keen on not talking 30 minutes, so set the opening to 9:15 or 9:20? :)

Comment (haard) [inline index.html]: Yeah, 30 minutes is insane. 10 or 15 minutes is enough!

**Comment by nailor** [comment]
I'm merging this in, please fix later to reflect reality. Hilton needs it asap

File: index.html (hunk 1 above)
Comment (anah) [inline index.html]: @nailor @haard Thanks! Fixed now: https://github.com/python-sverige/python-sverige.github.io/pull/56

**Pull request merged**